ark/repo/tsup.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from "tsup"
 
 export default defineConfig({
 	entry: ["main.ts"],
-	format: ["cjs"],
+	format: ["cjs", "esm"],
```

Comment:
I noticed that ESM modules were not being built without this change and the export paths in the package.json were pointing to nonexistent files, so I added this in. Please let me know if this should either be in a separate PR or not needed at all.